### ISSUE TITLE
[Neutral]2018-05-13

### ISSUE THREAD

**Issue by haoyang-udacity** [udacity/cvnd-waffle-issues#43]
The references code are for Tensorflow code and not Pytorch which looks clearly like a copy/paste of a portion of another nanodegree without taking care of this kind of things.\
There is no talk for CNN which is the object of this nanodegree (strides\, padding\, number of filters ...)
